src/utils/Api.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+
+const authorization_token = localStorage.getItem('authorization_token');
+
+const api_instance = axios.create({
+  baseURL: process.env.REACT_APP_BACKEND_URL,
+  headers: {
+    'Content-Type': 'application/json',
+    Authorization: authorization_token != null ? authorization_token : '',
+  },
+  responseType: 'json',
```

Comment:
This attribute `responseType` is not needed as it is a request that we are sending.